03-Control-Attestation/README.md

```diff
@@ -0,0 +1,337 @@
+
+### How to setup attestation repository in a project?
+
+In order to setup attestation repository inside a project, follow the below steps:
+
+1. Navigate to *Repos* section of the project.
+2. Create a new Git repository with the name 'ADOScannerAttestation'. Skip this step if this repository already exists.
+
+### How to setup host project to store attestation details for organization-specific controls?
+
+1. Host project to store attestation details for organization-specific controls can be set using a scan parameter named 'AttestationHostProjectName'. Before setting up the host project, ensure this project has 'ADOScanner_Attestation' repository setup.
+2.  **Attestation host project can be set only once and can't be updated later**.
+
+For e.g., to attest organization controls, run the command below:
+```PowerShell  
+#Set attestation host project and attest organization controls:
+$orgName = '<Organization name>'
+$hostProjectName = '<Name of the host project to store attestation details of org-specific controls>'
+  	
+Get-AzSKADOSecurityStatus -OrganizationName $orgName -AttestationHostProjectName $hostProjectName -ControlsToAttest NotAttested -ResourceTypeName Organization  
+
+```
+
+----------------------------------------------
+
+## Starting attestation
+      
+The AzSK.ADO scan cmdlets now support a new switch called ***ControlsToAttest***. When this switch is specified, 
+AzSK.ADO enters the attestation workflow immediately after a scan is completed. This ensures that attestation is done on the basis of the most current
+control status.
+
+All controls that have a technical evaluation status of anything other than 'Passed' (i.e., 'Verify' or 'Failed' or 'Manual' or 'Error') are considered 
+valid targets for attestation.
+
+> **Note**: Some controls are very crucial from security stand point and hence AzSK.ADO does not support attesting them.
+
+To manage attestation flow effectively, 4 options are provided for the *ControlsToAttest* switch to specify which subset of controls to target for attestation. These are described below:
+
+|Attestation Option|Description|
+|------------------|-----------|
+|NotAttested|Attest only those controls which have not been attested yet.|
+|AlreadyAttested|Attest those controls which have past attestations. To re-attest or clear attestation.|
+|All|Attest all controls which can be attested (including those that have past attestations).|
+|None|N/A.|
+
+The attestation feature internally stores attestation details in a repository called 'ADOScannerAttestation'. As a prerequisite, you need to create a repository with the same name. Attestation details of projects, builds, releases, service connections, agent pools and variable groups will be stored in the 'ADOScanner_Attestation' repository of the project these resources belong to.    
```

Comment:
"with the same name." or can be configured repo name in the org-policy if want to have repo with different name.